special-cat.md

```diff
@@ -0,0 +1 @@
+A special cat is a sweet and delicious pink squishy cat made out of marshmallows. Its meows are made of candy floss and its coat is spun sugar. If you ask nicely, he will make you a nutella crepe.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.